wsagent/che-core-api-project/src/main/java/org/eclipse/che/api/project/server/ProjectService.java

```diff
@@ -351,8 +361,16 @@ public void importProject(@ApiParam(value = "Path in the project", required = tr
                                                                      ServerException,
                                                                      NotFoundException,
                                                                      BadRequestException {
-        projectManager.importProject(path, sourceStorage, force,
-                                     () -> new ProjectImportOutputWSLineConsumer(path, workspace, 300));
+
```

Comment:
I believe it is not enough to rewrite the method here. The main idea that stands by this task is to replace all REST and old websocket calls by new JSON-RPC implementation. I mean we also need to replace the REST call itself (`@Path("/import/{path:.*}")`) by JSON-RPC call (`project/import` or something like that). So eventually we should not have any REST calls in this service, perhaps meanwhile we can have a separate JsonRpcProjectService to decouple REST and JSON-RPC logics. Do you see what I mean?